source/common/grpc/common.cc

```diff
@@ -53,11 +54,12 @@ void Common::chargeStat(const Upstream::ClusterInfo& cluster, const std::string&
   }
   cluster.statsScope()
       .counter(fmt::format("{}.{}.{}.{}", protocol, grpc_service, grpc_method,
-                           grpc_status->value().c_str()))
+                           grpc_status->value().getStringView()))
       .inc();
   uint64_t grpc_status_code;
+  std::string grpc_status_string(grpc_status->value().getStringView());
```

Comment:
consider use `absl::SimpleAtoi` instead of atoull (or implement StringUtil::atoull with absl::SimpleAtoi) so we can avoid this string allocation.

Reply:
Great idea - I will create an issue to follow up by either implementing `StringUtil::atoull` with `absl::SimpleAtoi` or converting call sites. I think there was only one spot in the code I found where the return value of `StringUtil::atoull` (which is C-style) was used.